docs/Commands.md

```diff
@@ -0,0 +1,36 @@
+# Commands
```

Comment:
Missing front matter (`--- … ---`) lines, copy and paste them from one of the other .md files.